scripts/build/termux_step_copy_into_massagedir.sh

```diff
@@ -1,12 +1,6 @@
 termux_step_copy_into_massagedir() {
-	local DEST="$TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX_CLASSICAL"
-	cd "$TERMUX_PREFIX_CLASSICAL"
-	mkdir -p "$DEST"
-	# Recreate each new directory and copy its metadata in one go
-	find . -path ./tmp -prune -o -type d -newer "$TERMUX_BUILD_TS_FILE" \
-		-exec mkdir -p "$DEST"/{} \; -exec chmod --reference="{}" "$DEST"/{} \;
-	# Copy only files modified during the build preserving original modes in order to massage them
-	# xargs is needed to avoid exceeding ARG_MAX
-	find . -path ./tmp -prune -o -newer "$TERMUX_BUILD_TS_FILE" \( -type f -o -type l \) \
-		-exec cp -P --preserve=all --parents -t "$DEST" {} +
+	mkdir -p "$TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX_CLASSICAL"
+	# Copy files changed during the build into massagedir in order to massage them
+	tar -C "$TERMUX_PREFIX_CLASSICAL" -N "$TERMUX_BUILD_TS_FILE" --exclude='tmp' -cf - . | \
```

Comment:
- 1.) are we already in `$TERMUX_PREFIX_CLASSICAL`?
- 2.) why get rid of `$DEST`?

Reply:
1. `-C` arg of `tar` performs directory changing automatically, no need for `cd.
2. Should we move path to variable if we use it only once?

Reply:
Derp mixed up `-C` and `-c`[^1].

[^1]: obligatory XKCD https://xkcd.com/1168/

Reply:
`$DEST` is still used twice.
Once in the `mkdir -p` once in the second `tar`

Reply:
I am not sure how I missed double usage. 
Fixed it.